cli_tools/gce_ovf_import/ovf_importer/ovf_importer_test.go

```diff
@@ -741,6 +742,50 @@ func TestPopulateMissingParametersInvalidZone(t *testing.T) {
 	assert.Equal(t, "europe-north1-b", params.Zone)
 }
 
+func TestHandleTimeoutSuccess(t *testing.T) {
+	mockCtrl := gomock.NewController(t)
+	defer mockCtrl.Finish()
+
+	mockLogger := mocks.NewMockLoggerInterface(mockCtrl)
+	mockLogger.EXPECT().Log("Timeout 0s exceeded, stopping workflow \"wf\"")
+
+	params := getAllInstanceImportParams()
+	params.Timeout = "0s"
+
+	oi := OVFImporter{Logger: mockLogger, params: params}
```

Comment:
The oi.Logger.Log doesn't impact the test - it has no return value, neither modifying anything. You may directly set a fake/empty logger. A mock logger is somehow overkill. But it's up to you.

Reply:
The logger output is used to validate the test using EXPECT()

Reply:
OK I thought you were validating w.CancelReason. It's OK to validate log as plus, on the other hand, isn't w.CnacelReason more important to validate?

Reply:
That is tested in workflow_test.go in the previous (Daisy related) PR. 

What would make sense to validate here is that w.CancelWithReason() actually gets called. This is partially done in ovf_importer_test.go by making sure w.Cancel channel is closed.